go.mod

```diff
@@ -1,4 +1,6 @@
-module github.com/joyent/triton-go
+module github.com/joyent/triton-go/v2
+
+go 1.14
```

Comment:
I don't think we want go 1.14 here (this means the minimum go version required... maybe?) - so perhaps just drop this line. I currently use 1.12 and everything builds/works fine (even with this go 1.14 line there).